velox/common/file/FileSystems.h

```diff
@@ -35,17 +35,29 @@ class FileSystem {
   FileSystem(std::shared_ptr<const Config> config)
       : config_(std::move(config)) {}
   virtual ~FileSystem() {}
+
   // Returns the name of the File System
   virtual std::string name() const = 0;
+
   // Returns a ReadFile handle for a given file path
   virtual std::unique_ptr<ReadFile> openFileForRead(std::string_view path) = 0;
+
   // Returns a WriteFile handle for a given file path
   virtual std::unique_ptr<WriteFile> openFileForWrite(
       std::string_view path) = 0;
 
   // Deletes the file at 'path'. Throws on error.
   virtual void remove(std::string_view path) = 0;
 
+  // Returns true if the file exists.
+  // Currently, this method will be used for testing, but we will need
```

Comment:
Is this comment for `exists` accurate? Based on Masha's comment below, should this be for `list` instead?

Reply:
Good catch